src/Components/Home/ListItem.js

```diff
@@ -10,16 +10,15 @@ function ListItem(prop) {
              <Row>
                 <Col>
                     {daysLeft == 0 ?  
-                        <Button variant="outline-dark" className="px-3"><i className="fas fa-shopping-cart"></i> Buy</Button>: 
+                        <Button variant="danger" className="px-3"><i className="fas fa-minus-circle"></i> Forgot</Button>: 
```

Comment:
I am not sure about this "danger" color. It's like we are telling the user that is a dangerous thing to forget about a product. It is indeed an irreversible action that will delete the product from the database. But this app is about to forget the products and buy less.
```suggestion
                        <Button variant="outline-danger" className="px-3"><i className="fas fa-minus-circle"></i> Forgot</Button>: 
```

Reply:
I agree that the 'Forget/Forego' button should look more rewarding to a user. I think it should be something that appears more like an achievement than anything negative so I would avoid red completely.

Reply:
And here also goes the name change, on this line.

Reply:
Changed to success-outline for now and changed to "forget"